dev/com.ibm.ws.security.social_fat.LibertyOP.backchannelLogout/publish/files/serversettings/socialClient_configTests.xml

```diff
@@ -368,6 +368,7 @@
 		tokenEndpoint="https://localhost:${bvt.prop.security_1_HTTP_default.secure}/oidc/endpoint/OidcConfigSample_useLogoutTokenForAccess/token"
 		hostNameVerificationEnabled="false"
 		authFilterRef="useLogoutTokenForAccess_introspect_filter"
+		accessTokenSupported="true"
```

Comment:
This isn't a valid attribute for the `<oidcLogin>` element. Only the `<oauth2Login>` element supports the `accessTokenSupported` attribute.

Reply:
Removing the accessTokenSupported in a separate pull.